11-payment-encoding.md

```diff
@@ -212,6 +211,8 @@ A reader:
     - MUST use the `n` field to validate the signature instead of performing signature recovery.
   - if there is a valid `s` field:
     - MUST use that as [`payment_secret`](04-onion-routing.md#tlv_payload-payload-format)
+  - if the `c` field (`min_final_cltv_expiry`) is not provided:
+    - MUST use an expiry delta of at least 18 when making the payment
```

Comment:
The writer now MUST include a `min_final_cltv_expiry` `c` field ([L173](https://github.com/lightningnetwork/lightning-rfc/pull/785/files#diff-16621ac6a270af29f36f0a4f900955fcR173)), therefore the reader should reject if `c` is not provided. 

On the one hand, this suggestion makes the spec more consistent. On the other, it may be a non-backwards compatible change. But if the two parties use different defaults payments will fail to go through anyway (if I understand correctly).

Reply:
That's exactly why we kept that fallback to 18 when missing, we don't want to make a backwards-incompatible change here.